MiniApp/Classes/ui/MiniAppViewController.swift

```diff
@@ -1,7 +1,21 @@
 import Foundation
 import UIKit
 
-class MiniAppViewController: UIViewController {
+public protocol MiniAppUIDelegate: class {
+    func onError(error: Error)
+    func onSuccess()
+    func onForward()
+    func onBackward()
+    func onClose()
```

Comment:
If I understood well, this delegate will be triggered by MiniAppViewController? If it's the case, when making a delegate I prefer using the following syntax: 
```swift
public protocol MiniAppUIDelegate: class {
    func miniApp(_ viewController: MiniAppViewController, didLaunchWith config: MiniAppSdkConfig? = nil, error: Error? = nil)
    func miniApp(_ viewController: MiniAppViewController, shouldExecute action: MiniAppNavigationAction)
}
```
Of course, you would have to add the `.close` action to `MiniAppNavigationAction`. But that's just my personal opinion. What do you think? @corycaywood, @ts-leojoseph-r

Reply:
@Climbatize 
yeah + the delegate should be used when using the `launch` function. 
If the user wants to customize or take more stuff in their hands I've exposed a create function which allows to just override the vc and the functions without really need to implement the delegate.

Syntaxwise sure would allow easy access to the vc 👍